### PR TITLE
Add playtest-20230218 announcement

### PR DESCRIPTION
**Pull request by PunkPun**

### CODE REVIEW

File: news/_posts/2023-02-19-playtest-20230219.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+---
+kind: article
+title: "Playtest 20230219"
+author: "PunkPun"
+created_at: 2023-02-19 00:00 +0000
+disqus_id: "playtest-20221223"
```
Comment (penev92) [inline news/_posts/2023-02-19-playtest-20230219.md:6]: Is this international?

File: _config.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ permalink: /:categories/:title/index.html
 
 download_page_tags:
   release: "release-20210321"
-  playtest: "playtest-20230110"
+  playtest: "playtest-20230219"
```
Comment (Mailaender) [inline _config.yml:15]:
```suggestion
  playtest: "playtest-20230218"
```

https://github.com/OpenRA/OpenRA/releases/tag/playtest-20230218

File: news/_posts/2023-02-18-playtest-20230218.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+---
+
+<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
+<div style="margin: -10px 5px" markdown="1">
+
+After a month of testing we present [Playtest 20230218]({{ '/download' | relative_url }})!
+
+Notable changes include:
+
+* Fixed Windows requiring visual C++ redistributable
```
Comment (abcdefg30) [inline news/_posts/2023-02-18-playtest-20230218.md:17]:
```suggestion
* Fixed Windows requiring a Visual C++ redistributable
```

File: news/_posts/2023-02-18-playtest-20230218.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+* Fixed minimap videos crashing
+* Polished Dune 2000 encyclopedia
```
Comment (abcdefg30) [inline news/_posts/2023-02-18-playtest-20230218.md:19]:
```suggestion
* Polished the Dune 2000 encyclopedia
```

File: news/_posts/2023-02-18-playtest-20230218.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+* Fixed MacOS missing the utility tool executable
+
+For more information see the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog).
```
Comment (abcdefg30) [inline news/_posts/2023-02-18-playtest-20230218.md:22]:
```suggestion
For more information see the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/bab2c14140d2db4819a59f742634a7ddd23c1cad).
```

File: news/_posts/2023-02-18-playtest-20230218.md (hunk 3 above)
Comment (PunkPun) [reply news/_posts/2023-02-18-playtest-20230218.md]: mention the year so I could use "the"

**Pull request merged**